incubator/jaeger/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.4.1
```

Comment:
since you are updating the jaeger image as well you need to update here to 1.8.2

Reply:
yes. Thanks for the review. I will send a new update.

Reply:
While we are on it, any suggestions on why lint is failing? I'm able to add incubator charts successfully via `helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator` but the CI does not like it

Reply:
lets see if this happens when you push the update